beacon-chain/sync/initial-sync/blocks_fetcher_test.go

```diff
@@ -98,20 +99,20 @@ func TestBlocksFetcher_InitStartStop(t *testing.T) {
 }
 
 func TestBlocksFetcher_RoundRobin(t *testing.T) {
-	blockBatchLimit := uint64(flags.Get().BlockBatchLimit)
-	requestsGenerator := func(start, end uint64, batchSize uint64) []*fetchRequestParams {
+	blockBatchLimit := types.Slot(flags.Get().BlockBatchLimit)
```

Comment:
Should this be a slot?

Reply:
Nope, it should be number (I just use it in many places below as slot). Will change to number.